app/src/main/res/layout/fragment_loan_account_summary.xml

```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ This project is licensed under the open source MPL V2.
+  ~ See https://github.com/openMF/self-service-app/blob/development/LICENSE.md
```

Comment:
Change ```development``` with ```master``` because right now It is not in master branch but soon we will release and will merge in master branch.